src/Microsoft.Health.Encryption/Customer/Health/CustomerKeyHealth.cs

```diff
@@ -0,0 +1,30 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Core.Features.Health;
+
+namespace Microsoft.Health.Encryption.Customer.Health;
+
+/// <summary>
+/// The health of the encryption key provided by the customer
+/// </summary>
+public class CustomerKeyHealth
+{
+    /// <summary>
+    /// Specifies if the resource is healthy or not
```

Comment:
You don't need to fix this, but typically in the base class libraries (BCL) they use the phrases `Gets [or sets]...` for their XML doc